test/system/helpers.bash

```diff
@@ -1,3 +1,4 @@
+#!/bin/bash
```

Comment:
This is not a good idea: it might give a reader the mistaken impression that `helpers.bash` is a runnable script. It is not; anyone familiar with BATS will know this, but other will not and might waste time trying to understand its usage. The lack of a shebang here is very deliberate. Are there conventions or directives that can tell `pre-commit` to not warn here?

Reply:
I think that adding any kind of shebang would avoid the error. Mainly the logic inside pre-commit is simple: .sh .bash are likely scripts, so they need execute bit and a shebang line. What is inside the shebamg line is mostly up to us.

Excludes are possible but I am doing my best to avoid them as they are not easy to maintain (a single regex for all, not a list). Better to find local workaround.

Reply:
I did not explain myself correctly. What I mean is, a shebang may cause confusion in a future maintainer: they might think that this is a runnable script. This has the potential to waste time.

Since there doesn't seem to be a clean exclusion mechanism, please change it to something like:
```
#!/bin/bash --help "This is a BATS helper; IT IS NOT AN EXECUTABLE SCRIPT"
```